### ISSUE TITLE
Attribute “href” is not required for element “a”

### ISSUE THREAD

**Issue by toptalo** [validator/validator#867]
Get an error: "Element “a” is missing required attribute “href”"

But [current specification](https://www.w3.org/TR/html52/textlevel-semantics.html#the-a-element) allows use element `a` without attribute `href`:

> If the `<a>` element has no `href` attribute, then the element represents a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element’s contents.

**Comment by sideshowbarker** [comment]
Please provide either a minimal document that causes the checker to emit the *"Element “a” is missing required attribute “href”"* message, or else a URL for an existing document that causes the checker to emit that message.

The checker won’t emit any error for the following minimal case:

```html
<!DOCTYPE html>
<html lang="">
<title>Test</title>
<a></a>
```

So, does the document that’s causing the checker to show you the *"Element “a” is missing required attribute “href”"* message have additional attributes? An `itemprop` attribute?

**Comment by toptalo** [comment]
Yes it has `itemprop="name"`

**Comment by sideshowbarker** [comment]
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element:names:-the-itemprop-attribute

> If the `itemprop` attribute is specified on an `a` element, then the `href` attribute must also be specified.

**Comment by toptalo** [comment]
Hmmmm, thanks! 👍
